widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetController.kt

```diff
@@ -35,8 +35,7 @@ internal class EntryWidgetController @JvmOverloads constructor(
             EntryWidgetContract.ItemType.LoadingState,
             EntryWidgetContract.ItemType.LoadingState,
             EntryWidgetContract.ItemType.LoadingState,
-            EntryWidgetContract.ItemType.LoadingState,
-            EntryWidgetContract.ItemType.ProvidedBy
+            EntryWidgetContract.ItemType.LoadingState
```

Comment:
Does it makes sense to add the white label item logic here, to keep it together with all loadingState items?

Reply:
I decided to keep this logic in one place in a sense of clarity so that it can be applied to all possible cases.